### PR TITLE
VizTooltip: Cleanup

### PR DESCRIPTION
**Pull request by leeoniya**
renames some props and types, removes some layers, simplifies the structure. hopefully more uniform and straightforward.

should be no functional changes.

### CODE REVIEW

File: packages/grafana-ui/src/components/VizTooltip/VizTooltipContent.tsx [hunk 1]
```diff
@@ -27,31 +32,20 @@ export const VizTooltipContent = ({ contentLabelValue, customContent, isPinned,
 
   return (
     <div className={styles.wrapper} style={scrollableStyle}>
-      <div>
-        {contentLabelValue.map((labelValue, i) => {
```
Comment (leeoniya) [inline packages/grafana-ui/src/components/VizTooltip/VizTooltipContent.tsx:31]: `contentLabelValue` was not used anywhere in the code

File: packages/grafana-ui/src/components/VizTooltip/VizTooltipHeader.tsx [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
-export const VizTooltipHeader = ({ headerLabel, keyValuePairs, customValueDisplay, isPinned }: Props) => {
+export const VizTooltipHeader = ({ item, isPinned }: Props) => {
   const styles = useStyles2(getStyles);
 
+  const { label, value, color, colorIndicator } = item;
+
   return (
     <div className={styles.wrapper}>
-      <HeaderLabel headerLabel={headerLabel} isPinned={isPinned} />
-      {customValueDisplay || <VizTooltipHeaderLabelValue keyValuePairs={keyValuePairs} isPinned={isPinned} />}
```
Comment (leeoniya) [inline packages/grafana-ui/src/components/VizTooltip/VizTooltipHeader.tsx:24]: `customValueDisplay` was nor used by anything

File: packages/grafana-ui/src/components/VizTooltip/VizTooltipHeader.tsx [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
 }
-export const VizTooltipHeader = ({ headerLabel, keyValuePairs, customValueDisplay, isPinned }: Props) => {
+export const VizTooltipHeader = ({ item, isPinned }: Props) => {
   const styles = useStyles2(getStyles);
 
+  const { label, value, color, colorIndicator } = item;
+
   return (
     <div className={styles.wrapper}>
-      <HeaderLabel headerLabel={headerLabel} isPinned={isPinned} />
```
Comment (leeoniya) [inline packages/grafana-ui/src/components/VizTooltip/VizTooltipHeader.tsx:23]: this was simply wrapping `VizTooltipRow`

**Review by baldm0mma — APPROVED** [review]
LGTM. Works as expected locally.

**Pull request merged**